Formula/qrcp.rb

```diff
@@ -0,0 +1,56 @@
+class Qrcp < Formula
+  desc "Transfer files to and from your computer by scanning a QR code"
+  homepage "https://claudiodangelis.com/qrcp"
+  url "https://github.com/claudiodangelis/qrcp/archive/0.6.1.tar.gz"
+  sha256 "a42e6540b0293138b4cf5de2f0d8a988303b289c2eb012a67a80b3527c3c8c5f"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    # QR for http://localhost:8080/send/testing
+    qr_code = <<~EOS
+      █████████████████████████████████████
+      █████████████████████████████████████
+      ████ ▄▄▄▄▄ █▀█▄██▀▄▄██▀███ ▄▄▄▄▄ ████
+      ████ █   █ ██ █  ▀▄██ ▄▀▄█ █   █ ████
+      ████ █▄▄▄█ █▄███▀▄ █▀▀▄▀▄█ █▄▄▄█ ████
+      ████▄▄▄▄▄▄▄█▄█ █ █▄▀ █ █▄█▄▄▄▄▄▄▄████
+      ████▄▀ █▄█▄██▄▄▄█▀█▄▄█▄▀▄▀▀█▄▀█▄▀████
+      ████▀██  ▄▄▄▀█▄█ ▀ ▀▀ █▀▄▀██ ▀▄ ▄████
+      ████▄ ▄▄█▄▄▄ █▀▀▄█  ▄▄█▄▄  ██ ▀▄ ████
+      █████▄█  ▀▄▀▀▄█  ▄█▄ ████▄▀ ▄▀█ ▄████
+      █████▄▄▀██▄▀ █▀▀█▀█▄ ▄▄▀█▄ ███▀▄ ████
+      ████▄▀█▀▄ ▄ ▀▀██▄▀ █ █▀▀▄█  █ █ ▄████
+      ████▄█▄█▄█▄▄▀▄▀▀▄█ ▄▄███ ▄▄▄ █▀▀▀████
+      ████ ▄▄▄▄▄ ███▄▀ ▄█  ██▀ █▄█ ▄█▄▄████
+      ████ █   █ █▄▀███▀█ ▄▀▄▀ ▄▄  █▀█▄████
+      ████ █▄▄▄█ █▄▄▀ █▄▀▄ █▀█▀▄█▄ █▄▀▄████
+      ████▄▄▄▄▄▄▄█▄█▄██▄▄▄▄▄██▄█▄▄███▄▄████
+      █████████████████████████████████████
+      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+    EOS
+
+    server_url = "http://localhost:8080/send/testing"
+    test_file = testpath/"test.txt"
+    received_file = testpath/"test_received.txt"
+    test_file_content = "Hello there, big world"
+    test_file.write test_file_content
+
+    begin
+      server = IO.popen([bin/"qrcp", "-i", "any", "-d", "localhost", "-p", "8080", "--path", "testing", test_file])
+      sleep 1
+
+      # User-Agent header needed in order for curl to be able to receive file
+      system "curl", "-H", "User-Agent: Mozilla", "-o", received_file, server_url
+
+      assert_match Regexp.new(qr_code), server.read
+      assert_equal shell_output("cat #{test_file}"), shell_output("cat #{received_file}")
```

Comment:
No need to write the expected value to a file then.

Reply:
Good point, no clue why I was doing that.

I removed the QR code validation, as it will change with varying port.